webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -1598,8 +1599,8 @@ class StringFormatter private (
   private def makeArkUrl(
     projectID: String,
     resourceID: String,
-    maybeValueUUID: Option[UUID] = None,
-    maybeTimestamp: Option[Instant] = None
+    maybeValueUUID: Option[UUID],
+    maybeTimestamp: Option[Instant]
```

Comment:
Not opposed, just wondering: Why remove the default values?

Reply:
They were unused. I.e. on every call side the parameter was provided.

Reply:
makes sense, then